data-conversion/src/main.rs

```diff
@@ -252,3 +267,43 @@ fn fixed_to_binary(
 
     Ok(())
 }
+
+fn binary_to_fixed(
+    binary_string: &str,
+    filepath_send: &mut File,
+    exp_str: &str,
+) -> io::Result<()> {
+    // Create an array with the elements of fixed_string
+    let words: Vec<&str> = binary_string.split_whitespace().collect();
+    let binary_str: &&str =
+        words.first().unwrap_or(&"There is not a binary number");
```

Comment:
Maybe this needs further commentary about why there is whitespace in the string at all? I guess this is more about what the arguments to the function should be: what does whitespace "mean" in the input string?